saleor/graphql/order/mutations/order_note_update.py

```diff
@@ -0,0 +1,58 @@
+import graphene
+
+from ....core.tracing import traced_atomic_transaction
+from ....order import OrderEvents, events, models
+from ....permission.enums import OrderPermissions
+from ...app.dataloaders import get_app_promise
+from ...core import ResolveInfo
+from ...core.descriptions import ADDED_IN_314, PREVIEW_FEATURE
+from ...core.doc_category import DOC_CATEGORY_ORDERS
+from ...core.types import OrderError
+from ...plugins.dataloaders import get_plugin_manager_promise
+from ..types import Order, OrderEvent
+from .order_note_common import OrderNoteCommon
+from .utils import get_webhook_handler_by_order_status
+
+
+class OrderNoteUpdate(OrderNoteCommon):
+    order = graphene.Field(Order, description="Order with the note updated.")
+    event = graphene.Field(OrderEvent, description="Order note updated.")
+
+    class Arguments(OrderNoteCommon.Arguments):
+        id = graphene.ID(
+            required=True,
+            description="ID of the note.",
+            name="note",
+        )
+
+    class Meta:
+        description = "Updates note of an order." + ADDED_IN_314 + PREVIEW_FEATURE
```

Comment:
I think the scope of 3.14 is closed. We could release it in 3.15.

```suggestion
        description = "Updates note of an order." + ADDED_IN_315 + PREVIEW_FEATURE
```